src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -168,10 +327,84 @@ export default class ManualBusinessInfo extends Vue {
     this.active = false
   }
 
+  /** When Jurisdiction value changes */
+  onJurisdictionChange (jurisdiction: any): void {
+    this.jurisdiction = jurisdiction
+    this.isMrasJurisdiction = MrasJurisdictions.includes(
+      this.jurisdiction?.text.toLowerCase()
+    )
+    if (jurisdiction?.group === 0) {
+      this.business.homeJurisdiction.country = JurisdictionLocation.CA
+      if (jurisdiction.value === JurisdictionLocation.FD) {
+        this.business.homeJurisdiction.region = 'Federal'
+      } else {
+        this.business.homeJurisdiction.region = jurisdiction.value
+      }
+    }
+
+    if (jurisdiction?.group === 1) {
+      this.business.homeJurisdiction.country = JurisdictionLocation.US
+      this.business.homeJurisdiction.region = jurisdiction.value
+    }
+
+    if (jurisdiction?.group === 2) {
+      this.business.homeJurisdiction.country = jurisdiction.value
+      this.business.homeJurisdiction.region = ''
+    }
+
+    this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Home jurisdiction is required'
```

Comment:
Oh, so you do use the jurisdiction component details section (the whitespace below it).

But, does your flow allow for jurisdiction to every be empty? This code may not be needed.

Reply:
Yes. Initially it is empty as user selects the Jurisdiction

Reply:
OK, looks good. Resolving.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/2fdb3b74-e571-48d4-bd55-e39c416c1771)